mapdata/WarcraftLegacies/AbilityData/Skin/1297035329.json

```diff
@@ -8,16 +8,22 @@
     {
       "Id": 1718840929,
       "Type": 3,
-      "Value": "(Twilight DK)"
+      "Value": "(Scourge DK)"
```

Comment:
The Black Strike ability and its associated buff should be deleted since they're no longer used.

Reply:
Okay was not sure about this ill start this practice moving forward.

Reply:
Quite reasonable not to be sure! In this case I'm sure Black Strike is only used here.

Reply:
Black Strike is vanilla Black Arrow it seems and am unable to delete (I assume that's the reason).
Both buffs used are vanilla also.